CEN4910C_Project/main.py

```diff
@@ -222,7 +240,7 @@ def Neigh_names():
         searchable=True,
         placeholder="Select neighborhood",
         style={
-            "width": "210px",
+            "width": "150px",

```

Comment:
Dropdown menus were previously uniform in width at 210px; this update has widths of 150, 160, and 190px. Stylistically, which is preferred?

Reply:
I'm assuming the different sizes for the dropdowns are meant to accommodate for length of the text in them. That said, maybe it would be better for them all to be 190px?

Reply:
Yeah I changed them in order to fit the dropdown of the new color option, that was it, if you guys find a different combo works than that is fine

Reply:
Here it is in the branch currently, different widths and justify-content: space-around:
![image](https://user-images.githubusercontent.com/84196204/163581464-b958411a-7c03-4f4b-a582-fdcf42c4e58e.png)

Different widths and justify-content: space-evenly:
![image](https://user-images.githubusercontent.com/84196204/163581601-8d4e5527-1647-4195-ac74-a884f3bb09cf.png)


Here it is with all the dropdowns at 190px and justify-content: space-around
![image](https://user-images.githubusercontent.com/84196204/163580731-c5f15efa-fdb1-4474-a9ca-30962bdb1d3f.png)

Here is all at 190px and justify-content: space-evenly:
![image](https://user-images.githubusercontent.com/84196204/163580966-91e9d607-49cf-4358-a89f-0083e8ab2aa0.png)

Reply:
Set style to uniform width (190px) and justify-content: space-evenly.